promq/autocomplete/earley/luthor.go

```diff
@@ -215,58 +214,58 @@ func createTokenFromItem(item promql.Item, offset int) Tokhan {
 	}
 }
 
-func mapParserItemTypeToTokhanType(item promql.Item) TokenType {
+func mapParserItemTypeToTokhanType(item parser.Item) TokenType {
```

Comment:
Tokhan :)

Reply:
hahaha whoops that wasn't supposed to make it out :)

Reply:
very creative :D